bms-monitor.js

```diff
@@ -1,18 +1,44 @@
 const {expect} = require('chai');
 
-function batteryIsOk(temperature, soc, charge_rate) {
+function checkBatteryTemperature(temperature) {
+    console.log("checkBatteryTemperature: ",temperature)
     if (temperature < 0 || temperature > 45) {

```

Comment:
Similar code is present for SOC check too. Code duplication can be reduced or avoided.